src/components/media-video.js

```diff
@@ -145,11 +145,14 @@ AFRAME.registerComponent("media-video", {
         // annoying "auto-pause" feature that forces one non-autoplaying video to play
         // at once, which will pause the videos for everyone in the room if owned.
         if (!isIOS && NAF.utils.getNetworkOwner(this.networkedEl) === "scene") {
-          setTimeout(() => {
-            if (NAF.utils.getNetworkOwner(this.networkedEl) === "scene") {
-              NAF.utils.takeOwnership(this.networkedEl);
-            }
-          }, 2000 + Math.floor(Math.random() * 2000));
+          setTimeout(
```

Comment:
If this is just a formatting change, it's probably best to revert it for now.

Reply:
This is from the eslint and prettier package bump with 'eslint --fix' doing the change. if I revert it, 'npm lint' will start failing.